commands/Settings.cs

```diff
@@ -104,7 +104,7 @@ public static async Task<Post> CreateFile(string title, DateTime date, string[]
                 // slug clean up for pesky words
                 var slug = title;
                 foreach (var keyword in Keywords) {
-                    slug = slug.Replace(keyword.Key, keyword.Value);
+                    slug = slug.Replace(keyword.Key, keyword.Value, StringComparison.OrdinalIgnoreCase);
```

Comment:
The `StringComparison.OrdinalIgnoreCase` from [here](https://github.com/khalidabuhakmeh/Frankenblog/pull/1/files#diff-0d3bb9c36f0f41819804816d6433c5505fb810bee403741a3ffa6cd73796fa7bL28) is not replacing the need to have it when you replace, it works if you would query in the Dictionary!